TinyMCE license key configuration docs (#2012)
plone/documentation: docs/backend/upgrading/version-specific-migration/upgrade-to-62.md

For example, `plone.locking` registers that `plone.locking.browser.info.pt` has a new location, `plone.app.layout.viewlets.locking.pt`.
You should rename your override to the new location if you no longer need compatibility with Plone 6.1 or earlier.
```

## TinyMCE 8 for Classic UI

> The WYSIWYG editor for Classic UI {glossary}`TinyMCE` was updated to Version 8.

Reviewer: ```suggestion
The {term}`WYSIWYG` editor for Classic UI, {glossary}`TinyMCE`, was updated to version 8.
```

> The {term}`WYSIWYG` editor for Classic UI, {glossary}`TinyMCE`, was updated to version 8.

Reviewer: Wow, I'm word blind! This will fix its display.

```suggestion
The {term}`WYSIWYG` editor for Classic UI, {term}`TinyMCE`, was updated to version 8.
```

> You are now able to provide your commercial license in the {menuitem}`TinyMCE controlpanel`.

Reviewer: ```suggestion
You're now able to enter your commercial license key in the {menuitem}`TinyMCE` control panel.
```

> See {doc}`/classic-ui/tinymce-customizations` for setup instructions.

Reviewer: Another I missed!

```suggestion
See {doc}`/classic-ui/tinymce-customization` for setup instructions.
```

> See this {doc}`guide </classic-ui/tinymce-customizations>` for setup instructions.

Reviewer: ```suggestion
See {doc}`/classic-ui/tinymce-customizations` for setup instructions.
```
